### PR TITLE
allow skipping the factory move command insertion

### PR DESCRIPTION
**Pull request by loveridge**
resolves https://github.com/beyond-all-reason/spring/issues/1538

From my testing, move command insertion doesnt seem necessary for any of the BAR factories and units

### CODE REVIEW

**Comment by sprunk** [comment]
could put the new bool check in line 438 as well to skip calculating the location

**Review by sprunk — APPROVED** [review]
Have't yet tested (do units still move away immediately if there's an active bugger-off area? do they stay if there is not?) but the code looks good.

File: rts/Sim/Misc/ModInfo.cpp [hunk 1]
```diff
@@ -205,6 +206,7 @@ void CModInfo::Init(const std::string& modFileName)
 		constructionDecay = constructionTbl.GetBool("constructionDecay", constructionDecay);
 		constructionDecayTime = (int)(constructionTbl.GetFloat("constructionDecayTime", 6.66) * GAME_SPEED);
 		constructionDecaySpeed = std::max(constructionTbl.GetFloat("constructionDecaySpeed", 0.03), 0.01f);
+		insertBuiltUnitMoveCommand = constructionTbl.GetBool("insertBuiltUnitMoveCommand", insertBuiltUnitMoveCommand);
```
Comment (sprunk) [inline rts/Sim/Misc/ModInfo.cpp:209]: I'd put this in the root-level table due to #1260 but this is already fine.

**Comment by lhog** [comment]
Thanks @loveridge

**Pull request merged**